itest/rust/src/register_tests/constant_test.rs

```diff
@@ -40,15 +40,35 @@ impl HasConstants {
     #[constant]
     #[cfg(all())]
     const CONSTANT_RECOGNIZED_WITH_SIMPLE_PATH_ATTRIBUTE_BELOW_CONST_ATTR: bool = true;
+
+    #[constant]
+    const CFG_REMOVES_CONSTANT: i64 = 5;
+
+    #[cfg(any())]
+    #[constant]
+    const CFG_REMOVES_CONSTANT: i64 = 10;
+
+    #[constant]
+    #[cfg(any())]
+    const CFG_REMOVES_CONSTANT: i64 = 10;
+
+    #[cfg(any())]
+    #[constant]
+    const CFG_REMOVES_CONSTANT_FFI_GLUE: bool = true;
+
+    #[constant]
+    #[cfg(any())]
+    const CFG_REMOVES_CONSTANT_FFI_GLUE: bool = true;
 }
 
 #[itest]
 fn constants_correct_value() {
-    const CONSTANTS: [(&str, i64); 4] = [
+    const CONSTANTS: [(&str, i64); 5] = [
         ("A", HasConstants::A),
         ("B", HasConstants::B as i64),
         ("C", HasConstants::C as i64),
         ("D", HasConstants::D as i64),
+        ("CFG_REMOVES_CONSTANT", HasConstants::CFG_REMOVES_CONSTANT),
```

Comment:
The name is a bit confusing, this looks like it tests for presence of a constant that was deliberately removed.

Maybe name them `ONLY_ACTIVE_CFG_CONSTANT` or so. And would be good to add a comment near the definition that the 3 are mutually exclusive.

Reply:
> The name is a bit confusing, this looks like it tests for presence of a constant that was deliberately removed.

True!

> Maybe name them ONLY_ACTIVE_CFG_CONSTANT or so. And would be good to add a comment near the definition that the 3 are mutually exclusive.

Okay. I chose `CFG_REMOVES_DUPLICATE_CONSTANT_DEF` as that felt a bit more appropriate, but I can change it to `ONLY_ACTIVE_CFG_CONSTANT` if that'd be better.